### PR TITLE
Install Composer from the official Docker image

### PR DESCRIPTION
**Pull request by tiagodevweb**
Congratulations on the repository, I noticed some other improvements besides this PR one but I'm waiting for your approval to send others. I'm also looking for production versions, how about scaling up this repository for that?

### CODE REVIEW

**Review by joseluisq — CHANGES_REQUESTED** [review]
The PR itself is not bad but the problem is that it does other things not related to the PR itself like changing config file paths.

**Comment by tiagodevweb** [comment]
alright, readjusted

**Review by joseluisq — CHANGES_REQUESTED** [review]

File: 7.4-fpm/Dockerfile [hunk 1]
```diff
@@ -287,10 +274,13 @@ RUN set -eux \
     && (find /usr/local/sbin -type f -print0 | xargs -n1 -0 strip --strip-all -p 2>/dev/null || true) \
     && true
 
+# Install Composer
+COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
```
[the remark this answers is not in the document]
Comment (tiagodevweb) [reply 7.4-fpm/Dockerfile]: I understand, it looks better! Adjusted.

**Review by joseluisq — APPROVED** [review]

**Comment by joseluisq** [comment]
Thanks 👍

**Pull request merged**